speaker/main.go

```diff
@@ -103,31 +132,41 @@ type controller struct {
 	announced map[string]config.Proto // service name -> protocol advertising it
 }
 
-func newController(myIP net.IP, myNode string, noARP bool) (*controller, error) {
-	var arpCtrl *arpController
-	if !noARP {
-		arpAnn, err := arp.New(myIP)
+type controllerConfig struct {
+	IP4    net.IP
+	IP6    net.IP
+	MyNode string
+
+	DisableARP bool
```

Comment:
Please add a comment that this is for testing only. I do still want to remove it at some point.